jenkins/bootstrap.py

```diff
@@ -478,6 +478,15 @@ def find_version(call):
         ]
         return call(cmd, output=True).strip()
 
+    # Make up some tag instead of unknown
+    cmd = ['git', 'describe', '--always', '--tags', '--abbrev=14']
+    try:
+        tag = call(cmd, output=True)
+        if tag:
+            return '+%s' % tag.strip()
```

Comment:
gubernator always look for k/k and will not anyway...

we can somehow append the repo name here and try to fix the link?